db/commitlog/commitlog.cc

```diff
@@ -214,6 +215,16 @@ struct db::commitlog::entry_writer {
      */
     virtual size_t size(segment&, size_t) = 0;
 
+    virtual size_t frag_offset(size_t) const {
+        return 0;
+    }
+    virtual size_t frag_remaining(size_t) const {
+        return 0;
+    }
+    virtual uint32_t frag_sequence_id(size_t) const {
+        return 0;
+    }
```

Comment:
Please add documentation. What's the size_t? I assume the entry number?

Reply:
Yes. Same as all other in the type. Can add name now, maybe make it a typedef later.

Reply:
Added comments.